Documentation/ApiOverview/CachingFramework/Architecture/Index.rst

```diff
@@ -245,9 +245,9 @@ Cache API
 
 The caching framework architecture is based on the following classes:
 
-- **\\TYPO3\\CMS\\Core\\Cache\\Frontend\\FrontendInterface**: Main interface to handle cache entries of a specific cache.
+- `\\TYPO3\\CMS\\Core\\Cache\\Frontend\\FrontendInterface`: Main interface to handle cache entries of a specific cache.
```

Comment:
```suggestion
- :php:`\TYPO3\CMS\Core\Cache\Frontend\FrontendInterface`: Main interface to handle cache entries of a specific cache.
```
In literal blocks the content is treated as plain text, therefore backslashes don't have to be escaped as they were in normal text.